### PR TITLE
Update troubleshooting.markdown

### PR DESCRIPTION
**Pull request by Nils154**
added suggestion to increase swap memory if your system freezes during first run

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Nils154, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16227).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/installation/troubleshooting.markdown [hunk 1]
```diff
@@ -43,3 +43,21 @@ For `iptables` systems (was the default for older distributions):
 iptables -I INPUT -p tcp --dport 8123 -j ACCEPT
 iptables-save > /etc/network/iptables.rules  # your rules may be saved elsewhere
 ```
+
+### System freezes
+
+Upon the first run or after an upgrade, Home Assistant uses a lot of resources to (re)compile all the integrations.
```
Comment (frenck) [inline source/_docs/installation/troubleshooting.markdown:50]: I think we should make very clear this is for a specific installation type.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/installation/troubleshooting.markdown (hunk 1 above)
Comment (Nils154) [reply source/_docs/installation/troubleshooting.markdown:50]: I thought it was pretty general, any system can run out of memory. Do you think it specifically only applies to a Raspberry Pi 2?

Comment (frenck) [reply source/_docs/installation/troubleshooting.markdown:50]: No, it is not generic, as we pre-build and pre-compile all dependencies (Python wheels) for all supported architectures and most installation types.

Comment (Nils154) [reply source/_docs/installation/troubleshooting.markdown:50]: I see your point. Added a line.

**Review by frenck — APPROVED** [review]
Thanks, @Nils154 👍

**Pull request merged**